### ISSUE TITLE
The Meaning of Total Variation Loss

### ISSUE THREAD

**Issue by Xinyu-Li-123** [gordicaleksa/pytorch-neural-style-transfer#11]
I found an explanation of the total variation loss from the [tutorial](https://www.tensorflow.org/tutorials/generative/style_transfer#total_variation_loss) on the official website of TensorFlow. It says that total variation loss is a regularization term that can decrease the high-freqency artifacts of the output image. Neural Style Transfer, without total variation loss, tend to increase the high frequency components of the content image. Maybe we can add this to the Readme file
